app/javascript/react/screens/App/common/forms/TextInputWithCheckbox.js

```diff
@@ -37,18 +43,11 @@ class TextInputWithCheckbox extends React.Component {
           </div>
         </Form.ControlLabel>
         <div className="col-md-2">
-          <div className="input-group">
-            <input
-              type="text"
-              className="form-control"
-              name={id}
-              id={id}
-              defaultValue={value}
-              readOnly={!this.state.inputEnabled}
-              onChange={event => onChange(event.target.value)}
-            />
-            <div className="input-group-addon postfix-label">{postfix}</div>
-          </div>
+          <InputGroup>
+            <FormControl type="text" name={id} id={id} readOnly={!this.state.inputEnabled} {...input} />
+            <InputGroup.Addon>{postfix}</InputGroup.Addon>
```

Comment:
I haven't had a chance yet to test this locally, but based on your screenshots it doesn't look like this postfix is rendering anymore?

Reply:
Yup, those labels are being rendered correctly (I forgot to update screenshots in the initial comment).